src/Cardano/Shell/Features/Logging.hs

```diff
@@ -16,35 +25,13 @@ import           Cardano.Shell.Types (CardanoConfiguration, CardanoEnvironment,
 -- Loggging feature
 --------------------------------------------------------------------------------
 
---------------------------------
--- Exceptions
---------------------------------
-
-data LoggingException
```

Comment:
Hm, okay. A bit strange that you don't have any top-level exceptions that you want to notify about.
What happens if you want to read a file and it's not there?